web3-dev/smart-contracts/autonomous-smart-contract.rst

```diff
@@ -3,17 +3,17 @@
 Autonomous Smart Contract
 =========================
 
-In smart-contract world of Massa, you can create smart-contracts that can be called in the future.
+In smart-contract world of Massa, you can create smart-contracts that can be triggered in the future.
 
 This system works by sending messages in the future specifying the address and the function you want to trigger.
 
 You need to use the ABI `sendMessage(...)` in your smart contract if you want to send a message to wakeup a function in the future.
 
 When the message is sent it will debit you the gas directly in this operation and store this message in a separate pool
-than operation one called async pool.
+than the operation one called async pool.
 
 This pool is totally deterministic because it's filled by information that come from operations included in blocks
```

Comment:
I think you are mixing general information (this sentence) with specific ones (ABI sendMessage above). I would restructure the file to separate things better.